src/libraries/Microsoft.Bcl.Numerics/tests/Microsoft.Bcl.Numerics.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <TargetFrameworks>$(NetFrameworkMinimum);$(NetCoreAppMinimum)</TargetFrameworks>
```

Comment:
`NetFrameworkMinimum` is empty when `DotNetBuildFromSource=true`, causing restore to fail for the project because the `TargetFramework` is empty.

Reply:
We should test against both TFMs anyway. Thanks for spotting.